### PR TITLE
Added new exercise 'wordy'

### PR DESCRIPTION
**Pull request by lathspell**
Another one. Based on the Java example.

### CODE REVIEW

File: exercises/wordy/src/main/kotlin/Wordy.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+object Wordy {
+
+    fun solve(category: YachtCategory, vararg dices: Int): Int {
```
Comment (eparovyshnaya) [inline exercises/wordy/src/main/kotlin/Wordy.kt]: should be something like 
```kotlin
fun asnwer (problem: String): Int
```

File: exercises/wordy/.meta/src/reference/kotlin/Wordy.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                when (operator) {
+                    "plus" -> answerRecursive(rest, result + operand)
+                    "minus" -> answerRecursive(rest, result - operand)
+                    "multiplied by" -> answerRecursive(rest, result * operand)
+                    "divided by" -> answerRecursive(rest, result / operand)
+                    else -> throw IllegalArgumentException("Unknown operator '$operator'")
+                }
+            }
+            input.isEmpty() -> result
+            else -> throw IllegalArgumentException("Cannot parse '$input'?")
```
Comment (eparovyshnaya) [inline exercises/wordy/.meta/src/reference/kotlin/Wordy.kt]: Not a big deal, just wondering, why the exception message asks )

Comment (lathspell) [reply exercises/wordy/.meta/src/reference/kotlin/Wordy.kt]: Just a typo, it's now ending with an exclamation mark in both exceptions.

File: exercises/wordy/src/main/kotlin/Wordy.kt (hunk 1 above)
Comment (lathspell) [reply exercises/wordy/src/main/kotlin/Wordy.kt]: done

**Pull request merged**